front/app/hooks/useInitiativesPermissions.ts

```diff
@@ -1,18 +1,18 @@
 import { useState, useEffect } from 'react';
-import { IInitiativeActionDescriptorName } from 'api/initiative_action_descriptors/types';
+import {
+  IInitiativeActionDescriptorName,
+  InitiativeDisabledReason,
+} from 'api/initiative_action_descriptors/types';
 import useInitativeActionDescriptors from 'api/initiative_action_descriptors/useInitiativeActionDescriptors';
 import { ActionPermission } from 'services/actionTakingRules';
 import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
 import useAuthUser from 'api/me/useAuthUser';
 
-export type IInitiativeDisabledReason = 'notPermitted';
-
 export default function useInitiativesPermissions(
   actionDescriptorName: IInitiativeActionDescriptorName
 ) {
-  const [actionPermission, setActionPermission] = useState<
-    ActionPermission<IInitiativeDisabledReason> | null | undefined
-  >(undefined);
+  const [actionPermission, setActionPermission] =
+    useState<ActionPermission<InitiativeDisabledReason> | null>(null);
```

Comment:
For the same reason as explained above, this typing is now incorrect: it suggests that it can have any `InitiativeDisabledReason` or `null`, but in fact it can only have `not_in_group` and `not_permitted`